irods/connection.py

```diff
@@ -181,14 +181,14 @@ def requires_cs_negotiation(self):
 
     @staticmethod
     def make_ssl_context(irods_account):
-        check_hostname = getattr(irods_account,'ssl_verify_server','hostname')
+        verify_server = getattr(irods_account,'ssl_verify_server','hostname')
         CAfile = getattr(irods_account,'ssl_ca_certificate_file',None)
         CApath = getattr(irods_account,'ssl_ca_certificate_path',None)
-        verify = ssl.CERT_NONE if (None is CAfile is CApath) else ssl.CERT_REQUIRED
+        verify = ssl.CERT_NONE if ((None is CAfile is CApath) or verify_server == 'none') else ssl.CERT_REQUIRED
```

Comment:
How do I interpret `None is CAfile is CApath`?

Reply:
I believe the following are all equivalent:
 - `None is CAfile is CApath`
 - `None is CAfile and None is CApath`
 - `CAfile is None and CApath is None`
 - `None == CAfile == CApath`
 - `None == CAfile and None == CApath`

And you can imagine the remaining combinations. :)

Reply:
Right.  Python is one of those weird languages where often `(a OP b OP c) `is not equivalent to either
```
(a OP b) OP c
```
or
```
a OP (b OP c)
```
For example, `a < b < c` means `(a < b) and (b < c)` ; this also ggoes for == , !=, etc

Reply:
Thanks guys. That line had me cross-eyed for a second.